src/main/java/github/weichware10/toolbox/ConfirmBox.java

```diff
@@ -0,0 +1,57 @@
+package github.weichware10.toolbox;
+
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+//How to Communicate between windows
+public class ConfirmBox {
+
+    static boolean answer;
+
+    public static boolean display(String title, String message){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.